packages/components/Badge/src/CounterBadge/CounterBadge.types.ts

```diff
@@ -31,6 +31,13 @@ export interface CounterBadgeProps extends Omit<BadgeCoreProps, 'appearance' | '
    */
   dot?: boolean;
 
+  /**
+   * If the text badge is intended to be used a discovery surface.
```

Comment:
This comment doesn't help me explain what "list" does. What's a discovery surface? Could we make this comment clearer ?

Reply:
Updated